guides/plugins/plugins/administration/adding-directives.md

```diff
@@ -0,0 +1,67 @@
+# Adding directives
+
+## Overview
+
+Directives in the Shopware 6 Administration are essentially the same as in any other Vue application.
+This guide will teach you how to register your directives on a global and on a local scope.
+
+Learn more about Vue Directives in their [documentation](https://vuejs.org/v2/guide/custom-directive.html)
```

Comment:
Maybe using an `embed` link? e.g.:

```suggestion
Learn more about Vue Directives in their documentation:

{% embed url="https://vuejs.org/v2/guide/custom-directive.html" %}
```

You might need to exclude that link from the link checker, just in case it's failing.